resources/views/dashboard/individual.blade.php

```diff
@@ -1,6 +1,8 @@
 <div class="with-sidebar with-sidebar:2/3">
     <x-quick-links>
-        @if ($user->individual->isConnector() || $user->individual->isConsultant())
+        @if ($user->individual->isConnector() ||
+            $user->individual->isConsultant() ||
+            $user->individual->inProgressContractedProjects)
```

Comment:
Likely should be the following:

```suggestion
            $user->individual->inProgressContractedProjects()->count())
```